src/components/skills-quiz/tests/SkillsCourses.test.jsx

```diff
@@ -130,25 +124,29 @@ const SkillsCoursesWithContext = ({
 
 describe('<SkillsCourses />', () => {
   test('renders the correct data', async () => {
-    let containerDOM = {};
-    await act(async () => {
-      const { container } = renderWithRouter(
-        <SkillsCoursesWithContext
-          index={testIndex}
-        />,
-      );
-      containerDOM = container;
-    });
+    const { container } = renderWithRouter(
+      <SkillsCoursesWithContext
+        index={testIndex}
+      />,
+    );
 
-    expect(screen.getByText(SKILLS_HEADING)).toBeInTheDocument();
-    expect(screen.getByAltText(TEST_PARTNER.name)).toBeInTheDocument();
+    await waitFor(() => {
+      expect(screen.getByText(SKILLS_HEADING)).toBeInTheDocument();
+      expect(screen.getByAltText(TEST_PARTNER.name)).toBeInTheDocument();
+    });
 
-    expect(containerDOM.querySelector('.search-result-card > a')).toHaveAttribute(
-      'href',
-      `/${TEST_ENTERPRISE_SLUG}/course/${TEST_COURSE_KEY}`,
-    );
-    expect(containerDOM.querySelector('p.partner')).toHaveTextContent(TEST_PARTNER.name);
-    expect(containerDOM.querySelector('.pgn__card-image-cap')).toHaveAttribute('src', TEST_CARD_IMG_URL);
+    // expect(screen.querySelector('.search-result-card > a')).toHaveAttribute(
```

Comment:
[curious].  A permanent reminder of a former test! Wondering if it needs to be removed?

Reply:
Ha! Just an uncaught reminder of needing to assert on `userEvent.click()` of the `Card` that `history.push` is being called properly. Great catch! Fixing now.